web/packages/shared/components/UnifiedResources/shared/CopyButton.tsx

```diff
@@ -60,20 +61,16 @@ export function CopyButton({
   }, []);
 
   return (
-    <HoverTooltip tipContent={copiedText}>
-      <ButtonIcon
-        setRef={copyAnchorEl}
-        size={0}
-        mr={mr}
-        ml={ml}
-        onClick={handleCopy}
-      >
-        {copiedText === copySuccess ? (
-          <Check size="small" />
-        ) : (
-          <Copy size="small" />
-        )}
-      </ButtonIcon>
-    </HoverTooltip>
+    <Box mr={mr} ml={ml}>
+      <HoverTooltip tipContent={copiedText}>
+        <ButtonIcon setRef={copyAnchorEl} size={0} onClick={handleCopy}>
```

Comment:
When I see that we decided to solve this by moving `ml` and `mr` from `ButtonIcon` to an extra `Box`, my first thought is doesn't this mean there are likely multiple other instances where this might be a problem? Or does it only affect `ButtonIcon` wrapped in `HoverTooltip` that also set a margin on `ButtonIcon`?

I suppose what I really mean by this is that it'd be good to know what was the cause of this spacing bug – was it a change in `CopyButton` or `HoverTooltip`? If in `HoverTooltip`, it'd be good to understand why this happen as other places are likely to suffer from this problem as well – and this might suggest that perhaps a long-term fix should be applied in `HoverTooltip`, not `CopyButton`.